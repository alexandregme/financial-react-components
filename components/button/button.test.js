import React from 'react';
import { shallow } from 'enzyme';
import Button from 'components/button';

describe('Button', () => {
    let mountedButton;
    let button;

    beforeEach(() => {
        mountedButton = shallow(<Button />);
        button = mountedButton.find('button');
    });

    it('renders without crashing', () => {
        shallow(<Button />);
        expect(mountedButton).toMatchSnapshot();
    });

    it('expect to have button tag with default value', () => {
        expect(button).toHaveLength(1);
        expect(button.text()).toEqual('button');
    });
});
