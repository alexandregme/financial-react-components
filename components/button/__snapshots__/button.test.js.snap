// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders without crashing 1`] = `
<button>
  button
</button>
`;
